platform/mds/mds/src/main/java/org/motechproject/mds/util/Constants.java

```diff
@@ -453,8 +453,10 @@ private AnnotationFields() {
         public static final String AUTO_GENERATED = "autoGenerated";
         public static final String AUTO_GENERATED_EDITABLE = "autoGeneratedEditable";
         public static final String SQL_QUERY = "javax.jdo.query.SQL";
-        public static final String MDS_TABLE_PREFIX = "MDS_";
-        public static final String MDS_DATABASE = "motech_data_services";
+        public static final String MDS_DATABASE_KEY = "mds.data.databaseName";
```

Comment:
I'm now wondering, maybe just have two datanucleus.properties files? datanucleus_data.properties and datanucleus_schema.properties. That would allow us to clean some code up.